well-known-bots.json

```diff
@@ -130,6 +200,7 @@
     "pattern": "Mediapartners \\(Googlebot\\)",
     "addition_date": "2017/08/08",
     "url": "https://support.google.com/webmasters/answer/1061943?hl=en",
+    "verification": [],
```

Comment:
I'm surprised this doesn't have verification

Reply:
Similar to the one above, they only document `Mediapartners-Google`. And I don't want to add verification to undocumented bots because I wouldn't know what mask to use.

Reply:
Please open an issue to look into this. We will be able to search requests arriving to see if it exists.

Reply:
https://github.com/arcjet/well-known-bots/issues/23